modules/ROOT/pages/visibility.adoc

```diff
@@ -0,0 +1,122 @@
+
+Follow these steps to configure your community for API visibility:
+
+. Create a new *ACM Community User Plus* profile based on the *Customer Community User Plus* profile:
+.. Open *Salesforce Setup* > *Profiles*.
+.. Select the *Customer Community User Plus* profile.
+.. Click *Clone*.
+.. Enter a name for the new profile that represents the use case, such as `External Developer`.
+.. Assign permissions that match the member user profile permissions described in the API Community Manager documentation section xref:create-community.adoc#configure-member-user-profile-permissions[Configure Member User Profile Permissions].
+. Configure your community to use the new *ACM Community User Plus* profile:
+.. Open *Setup* > *All Communities*.
+.. In the API Community Manager control panel, open *Community Administration* and open the *Members* tab.
+.. In the *Select Profiles* section, use the search dropdown to select *Customer*.
+.. Add the new *ACM Community User Plus* profile to the selected profiles.
+.. _Do not_ remove the *System Administrator* profile.
++
+If you remove the *System Administrator* profile, the system locks you out and removes all access to your community.
+.. Remove the existing *ACM Community User* profile.
+. Remove all profiles from your community sharing set:
+.. Open *Setup* > *Communities Settings*.
+.. In the *Sharing Sets* section, edit your community sharing set and remove all profiles from the *Selected Profiles*.
+.. If the *Selected Objects* in the sharing set include *CommunityApi*, remove *CommunityApi* from the *Selected Objects*.
+. If your community already has users, open *Setup* > *Users* and update each user to change the *User License* to *Customer Community Plus* and select the new *ACM Community User Plus* profile.
+. In the API Community Manager control panel, set the visibility of each of your APIs to *Members Only*.
+
+=== Define Member Groups (Public Groups)
+
+Use member groups, also known as public groups, to group members and then make individual APIs visible to the group. Visibility rules, also known as sharing settings, determine the APIs visible to a member group.
+
+To create a member group:
+
+. Open *Setup* > *Public Groups*.
+. Create a *Public Group*.
+. Enter a name, such as `External Developers`.
+. Click *Save*.
+
+To add members to a member group:
+
+. Open *Setup* > *Public Groups*.
+. Edit the group.
+. In the *Search* dropdown, select *Customer Portal Users*.
+. Add the members by moving them from *Available Members* to *Selected Members*.
++
+*Available Members* shows only users with the new *ACM Community User Plus* profile. If the user you want to add is not in *Available Members*, open the user record and verify the profile.
+. Click *Save*.
+
+=== Define Visibility Rules (Sharing Settings) for APIs
+
+Use visibility rules, also known as sharing settings, to define a set of conditions that determine if an API is visible to a specific member group.
```

Comment:
ditto re. defining synonyms at the outset and then using only one thereafter.

Reply:
Fixed.